src/main/java/org/folio/inventory/dataimport/handlers/actions/ReplaceInstanceEventHandler.java

```diff
@@ -413,6 +418,14 @@ private void getActualInstanceAndReInvokeCurrentHandler(Instance instance, Insta
       });
   }
 
+  private void setUpdatedBy(org.folio.rest.jaxrs.model.Record changedRecord, String userId) {
```

Comment:
@Maksat-Galymzhan  Maybe it would be better to move this method to the Util and re-use it in different handlers?

Reply:
done